plugins/tedge_apt_plugin/src/lib.rs

```diff
@@ -34,14 +34,24 @@ pub struct AptCli {
 #[derive(clap::Subcommand, Debug)]
 pub enum PluginOp {
     /// List all the installed modules
+    #[clap(group(clap::ArgGroup::new("filter_target").multiple(true)))]
     List {
         /// Filter packages list output by name
-        #[clap(long, short)]
+        #[clap(long, short, group = "filter_target")]
         name: Option<String>,
 
         /// Filter packages list output by maintainer
-        #[clap(long, short)]
+        #[clap(long, short, group = "filter_target")]
         maintainer: Option<String>,
+
+        /// Filtering mode
+        #[clap(
+            long = "filter-mode",
+            requires = "filter_target",
+            default_value_t,
+            value_enum
+        )]
+        filter_mode: FilterMode,
```

Comment:
I don't understand the purpose of this `--filter-mode` option. Aren't `apt.include.name` and `apt.exclude.name` enough do define filtering expectations?

Reply:
Yes this is the same point I raised in my review. There is no point having the `--filter-mode` flag. The config values should just be mapped to flags, this would allow the user to test both inclusive and exclusive filters at the same time (which is closer to what actual behaviour when run by the tedge-agent).